src/pc/lua/smlua_functions.c

```diff
@@ -946,6 +946,69 @@ int smlua_func_collision_find_surface_on_ray(lua_State* L) {
     return 1;
 }
 
+  ////////////////
+ // graph node //
+////////////////
+
+typedef struct { s16 type; u16 lot; } GraphNodeLot;
+static GraphNodeLot graphNodeLots[23] = {
+    { GRAPH_NODE_TYPE_ANIMATED_PART, LOT_GRAPHNODEANIMATEDPART },
```

Comment:
these lot constants are autogenerated and shouldn't be used directly in C. those constants should be made non autogenerated